src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TextUi;
+import duke.task.TaskList;
+
+/**
+ * The ByeCommand class is used when an exit command is given.
+ */
+public class ByeCommand extends Command {
+
+    /**
+     * Constructs a bye Command, which sets the boolean value isExit to true.
+     */
+    public ByeCommand() {
+        super("");
+        this.isExit = true;
```

Comment:
Try to change parent class attributes with a setter or an overloaded constructor. Direct overwriting from the child class kinda defeats the encapsulation aspect of OOP.